src/process.rs

```diff
@@ -109,11 +109,12 @@ impl Processes {
             Some(pid) => {
                 let pid_str = pid.to_string();
                 let args = vec!["-9", &pid_str];
-                Command::new("kill")
-                    .args(args)
-                    .spawn()
-                    .expect("kill command failed to start");
-                info!("{} stopped", name);
+                let status = Command::new("kill").args(args).status();
+                if status.is_ok() {
+                    info!("kill {} {} exit with {}", name, &pid_str, status.unwrap());
+                } else {
+                    info!("kill {} {} failed", name, &pid_str);
+                }
```

Comment:
Should check if the exit status is success? https://doc.rust-lang.org/std/process/struct.ExitStatus.html
``` rust
match status {
    Ok(exit_status) if exit_status.success() => {
        info!("kill {} {} exit with {}", name, &pid_str, exit_status.code().unwrap_or(0));
    },
    _ => info!("kill {} {} failed", name, &pid_str);
}
```